.gitignore

```diff
@@ -1,2 +1,6 @@
 /.idea/
 .DS_Store
+
+#pnpm
+site/pnpm-lock.yaml
+#pnpm end
```

Comment:
I think these changes should be kept only on your side since we are neither using pnpm, nor we have suggested it anywhere.

Reply:
I don't think it's a big deal to include them. That can make it easier for use to exclude the file every time we want to commit.

Reply:
Why would you need to exclude it every time you commit? Just don't include it, that's it.

Reply:
Correct. Everytime i want to commit, i must exclude them manually.

Reply:
What? How is that possible?

Reply:
As far as I know, Git does not add any changes, from any file, unless you explicitly do it with `git add`. Do you always do `git add -A`?

Reply:
Yup. I always do git add -A rather than add every files manually. But, it is not big deal thou. Should i remove this line?

Reply:
I don’t mind if it stays there. I was just curious about it. It doesn’t change anything for the end users.

Reply:
Trust me. It will help me a lot. 😁

Reply:
Well, let’s keep it then :)